src/test/java/io/redisearch/client/ClientTest.java

```diff
@@ -992,4 +992,25 @@ public void testConfig() throws Exception {
 
         assertFalse(cl.setConfig(ConfigOption.ON_TIMEOUT, "null"));
     }
+
+    @Test
+    public void testAlias() throws Exception {
+        Client cl = getClient();
+        cl._conn().flushDB();
+
+        Schema sc = new Schema().addTextField("field1", 1.0);
+        assertTrue(cl.createIndex(sc, Client.IndexOptions.defaultOptions()));
+
+        assertTrue(cl.addAlias("TEST1"));
```

Comment:
@dengliming can you please add verification after each step that the alias really alias the index?
e.g. by calling info()

Reply:
That should add a field `alias` to `Query` to support custom alias search. What do you think?

Reply:
```java
    @Test
    public void testAlias() throws Exception {
        Client cl = getClient();
        cl._conn().flushDB();

        Schema sc = new Schema().addTextField("field1", 1.0);
        assertTrue(cl.createIndex(sc, Client.IndexOptions.defaultOptions()));
        Map<String, Object> doc = new HashMap<>();
        doc.put("field1", "value");
        assertTrue(cl.addDocument("doc1", doc));


        assertTrue(cl.addAlias("ALIAS1"));
        Client alias1 = getClient("ALIAS1");
        SearchResult res1 = alias1.search(new Query("*").returnFields("field1"));
        assertEquals(1, res1.totalResults);
        assertEquals("value", res1.docs.get(0).get("field1"));
        
        assertTrue(cl.updateAlias("ALIAS2"));
        Client alias2 = getClient("ALIAS2");
        SearchResult res2 = alias2.search(new Query("*").returnFields("field1"));
        assertEquals(1, res2.totalResults);
        assertEquals("value", res2.docs.get(0).get("field1"));

        try {
            cl.deleteAlias("ALIAS3");
            Assert.fail("Should throw JedisDataException");
        } catch (JedisDataException e) {
            // Alias does not exist
        }
        assertTrue(cl.deleteAlias("ALIAS2"));
        try {
          cl.deleteAlias("ALIAS2");
          Assert.fail("Should throw JedisDataException");
      } catch (JedisDataException e) {
          // Alias does not exist
      }
    }
```

Reply:
@dengliming I think your original PR is enough just extend your test like the one I added